tests/testthat/test-get_summary.R

```diff
@@ -4,4 +4,200 @@
 #' T1.3 An error when `survfit_object` is a tibble
 #' T1.4 An error when `survfit_object` is a data.table
 #' T1.5 An error when `survfit_object` is a random object
-#' T1.6 An error when `survfit_object` is NULL
\ No newline at end of file
+#' T1.6 An error when `survfit_object` is NULL
+#' T2. The function accepts a survival object
+#' T2.1 An error when `statlist` is NULL
+#' T2.2. An error when `statlist` is a random string
+#' T2.3. The `strata` in the statlist of `get_summary` is the same as `strata` in the `survfit` object
```

Comment:
This is a bit vague.
Do you mean. When `statlist` = `strata` the output stores the strata of the `survfit` object in the column xxx?

+ same for other values